tools/lint/src/main/java/com/grab/lint/inspector/Inspector.kt

```diff
@@ -0,0 +1,172 @@
+package com.grab.lint.inspector
+
+import com.android.tools.lint.client.api.IssueRegistry
+import com.android.tools.lint.client.api.LintClient
+import com.android.tools.lint.detector.api.Issue
+import com.android.tools.lint.detector.api.Severity
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import java.io.File
+import java.io.InputStreamReader
+import java.net.MalformedURLException
+import java.net.URL
+import java.net.URLClassLoader
+import java.util.jar.Attributes
+import java.util.jar.JarFile
+
+class Inspector(
+    private val outPut: File,
+    private val lintChecks: List<File>?,
```

Comment:
Instead of making this null, can make utilize `default` from `clikt` and have an empty list.

https://github.com/grab/grab-bazel-common/blob/0a4069a4df5cd27115fd41623274a9f97657bcb0/tools/lint/src/main/java/com/grab/lint/LintBaseCommand.kt#L162